crt_portal/cts_forms/views.py

```diff
@@ -577,6 +578,7 @@ def post(self, request):
                 'selected_all': 'all' if selected_all else '',
                 'ids': ids,
                 'ids_count': ids_count,
+                'show_warning': ids_count > 15,
```

Comment:
Small opportunity here to move this into a separate function and/or setting so to avoid hardcoding `15` in two separate places in the same view.